Reflection/Internal/RSGroundObject.simba

```diff
@@ -70,6 +80,141 @@ begin
   RFreeObject(R_EIOS, Items);
 end;
 
+(*
+RSGroundObject.GetTileByID
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Returns all tiles that contain a specific object ID.
+
+Example:
+
+  Writeln(RSGroundObject.GetTileByID(526));
+*)
+Function RSGroundObject.GetTileByID(itemID: Int32): Array of RSTile; static;
+var
+  Tile: RSTile;
+  Plane: Int32;
+  BaseX, BaseY: Int32;
+  MinX, MinY, MaxX, MaxY: Int32;
+  X, Y: Int32;
+  Player: RSPlayer;
+
+  Items: Pointer;
+  ItemsSize: SizeUInt;
+  NodeDeque: RSNodeDeque;
+  Head, Curr: RSNode;
+  Garbage: RSTYpeArray;
+
+  ID: Int32;
+begin
+  Player := RSPlayer.Me;
+  Tile := Player.Tile;
+  Player.Free;
+
+  BaseX := RSClient.BaseX;
+  BaseY := RSClient.BaseY;
+  Plane := RSClient.Plane;
+
+  MinX := BaseX;
+  MinY := BaseY;
+  MaxX := BaseX + 104;
+  MaxY := BaseY + 104;
+
+  ItemsSize := 0;
+  Items := RGetArray(R_EIOS, nil, ItemsSize, Client_GroundItems);
+  if ItemsSize = 0 then
+    Exit;
+
+  for Y := MinY to MaxY - 1 do
+  begin
+    for X := MinX to MaxX - 1 do
+    begin
+      NodeDeque.ref := RGetObjectArray3D(R_EIOS, Items, Plane, X - BaseX, Y - BaseY);
+      if NodeDeque.ref <> nil then
+      begin
+        Head := NodeDeque.Head;
+        Curr := Head.Next;
+        Garbage := [NodeDeque, Head, Curr];
+
+        while not RObjectsEqual(R_EIOS, Head.ref, Curr.ref) do
+        begin
+          ID := RGetInt(R_EIOS, Curr.ref, ITEM_ID);
+          if (itemID <> ID) then
+            break();
+
+          Result += [X, Y];
+
+          Curr := Curr.Next;
+          Garbage += Curr;
+        end;
+      end;
+    end;
+  end;
+
+  Garbage.Free;
+  RFreeObject(R_EIOS, Items);
+end;
+
+(*
+RSGroundObject.GetMSByID
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Returns the Mainscreen coordinates of a specific object ID.
+
+Example:
+
+   Mouse.Move(RSGroundObject.GetMSByID(526)[0]);
+*)
+Function RSGroundObject.GetMSByID(itemID: Int32): Array of RSTile; static;
```

Comment:
This should be in `Reflection/Objects.simba` instead of in `Reflection/Internal/RSGroundObject`.
The internal folder is for crucial functions that are `required` for the include to work. The outer folder is just for functions that are `convenient` for us to use :)